src/LogViewer/index.js

```diff
@@ -75,7 +75,7 @@ export default class LogViewer extends React.Component {
       parent.postMessage(JSON.stringify(qs), '*');
     }
 
-    history.pushState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
+    history.replaceState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
```

Comment:
Every time you highlight a different line, the state changes and records it in your history and you would be able to see it in your history browser
<img width="372" alt="screen shot 2017-01-10 at 2 10 59 pm" src="https://cloud.githubusercontent.com/assets/3766511/21820689/b431c77c-d73e-11e6-94ad-0a688abde504.png">

However, clicking back will not take you nowhere anymore. With `history.replaceState`, It will take you back to the previous page.